src/draw/drawMarker.ts

```diff
@@ -89,8 +89,8 @@ export function drawMarker(
   }
   if (shape === 'square') {
     const origin = {
-      row: Math.floor(point.row - size / 2),
-      column: Math.floor(point.column - size / 2),
+      row: Math.round(point.row - size / 2),
```

Comment:
I would use size-1 for row and col, seems to make more sense to me
`row: Math.round(point.row - (size-1) / 2)`
You want that if the size is 1 the pixel is exactly at the specified position.